core/src/main/java/com/cloud/agent/api/CheckOnHostCommand.java

```diff
@@ -37,6 +37,7 @@ public CheckOnHostCommand(Host host) {
     public CheckOnHostCommand(Host host, boolean reportCheckFailureIfOneStorageIsDown) {
         super();
         this.host = new HostTO(host);
+        setWait(20);
```

Comment:
I have some issues with this. As `super()` is called wait is set explicitely to 0 (zero). Either we should not call `super()` but `this(host)` or not set the wait value here.

makes sense?

Reply:
makes sense